src/containerapp/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.3.32
```

Comment:
Do you sure you want to release a new version for containerapp? If not want to release a new version, just use 'Upcoming' rather than '0.3.32'